apps/backend/src/mcp/tools/device_management.py

```
@@ -911,7 +898,9 @@ async def get_device_summary(device: str, timeout: int = 30) -> dict[str, Any]:
     except Exception as e:
         logger.error(f"Error getting device summary for {device}: {e}")
         raise DatabaseOperationError(
-            operation="get_device_summary", details={"device": device, "error": str(e)}
+            message=f"Failed to get device summary: {str(e)}",
+            operation="get_device_summary", 
```

Ruff (0.12.2)

Trailing whitespace

Remove trailing whitespace

(W291)